Tests/NIOOpenSSLTests/OpenSSLIntegrationTest.swift

```diff
@@ -298,8 +304,8 @@ class OpenSSLIntegrationTest: XCTestCase {
         guard let fileName = fileName else {
             fatalError("couldn't make temp file")
         }
-        let tempFile = try fileName.withCString { ptr in
-            return try Posix.open(file: ptr, oFlag: O_RDWR | O_CREAT | O_TRUNC | O_CLOEXEC, mode: 0o644)
+        let tempFile = fileName.withCString { ptr in
```

Comment:
should we make the `Posix` methods `public`?

Reply:
Nah, I don't think so. They're mostly an internal implementation detail tbh.